configure.ac

```diff
@@ -932,6 +936,44 @@ if test -z "$enable_fast_no_strict_alignment" ; then
     AC_DEFINE(NEEDS_STRICT_ALIGNMENT,1,[Define if strict alignment memory access is required])
 fi
 
+if test "$enable_fast_avx_instr" = "yes" ; then
+    AC_CACHE_CHECK([whether -mavx is supported], pac_cv_found_avx,
+                   [PAC_C_CHECK_COMPILER_OPTION([-mavx],pac_cv_found_avx=yes,pac_cv_found_avx=no)],
+                   pac_cv_found_avx=no,pac_cv_found_avx=yes)
+    if test "$pac_cv_found_avx" = "yes" ; then
+        PAC_APPEND_FLAG([-mavx],[CFLAGS])
+        PAC_APPEND_FLAG([-mavx],[CXXFLAGS])
+        PAC_APPEND_FLAG([-mavx],[FFLAGS])
+        PAC_APPEND_FLAG([-mavx],[FCFLAGS])
+        PAC_APPEND_FLAG([-mavx],[WRAPPER_CFLAGS])
+        PAC_APPEND_FLAG([-mavx],[WRAPPER_CXXFLAGS])
+        PAC_APPEND_FLAG([-mavx],[WRAPPER_FFLAGS])
+        PAC_APPEND_FLAG([-mavx],[WRAPPER_FCFLAGS])
```

Comment:
are the Fortran and C++ flags necessary here? The usage is in C code. Does LDFLAGS need to be modified?

Reply:
You're probably right. I was being conservative early on, but this can probably be pruned now. I think I copied this code from here:

https://github.com/pmodels/mpich/blob/3e44a99995599c15e9c269f0f984c0b89833f7f7/configure.ac#L1328-L1360